hs/GraphResponse.hs

```diff
@@ -52,7 +52,9 @@ timetableTab =
 sideBar :: H.Html
 sideBar = do
     H.div ! A.id "sidebar" $ do
-        H.div ! A.id "fcecount" $ ""
+        H.div ! A.id "fce" $ do
+            H.div ! A.id "fcecount" $ ""
+            H.button ! A.id "reset" $ "Reset Graphs"
```

Comment:
No money (`$`) necessary. `$` is *similar* to using parentheses (however, *not* identical).

[Here's a good source on the matter](http://learnyouahaskell.com/higher-order-functions)
Scroll down to *"Function application with $"*.